.github/workflows/test.yml

```diff
@@ -9,6 +9,11 @@ jobs:
       matrix:
         python-version: ["3.5", "3.6", "3.7", "3.8", "pypy-3.7"]
         mongodb-version: ["4.4"]
+        pymongo-version: ["v3.11", "v4.0"]
+        # pymongo 4.0 does not support python 3.5
```

Comment:
Can you file a new issue to drop support for 3.5 if one doesn't exist already?

Reply:
Done.